### PR TITLE
Update Vibrato to 0.3.1

### PR DESCRIPTION
**Pull request by kampersanda**
This PR updates the version of Vibrato to v0.3.1 and removes the submodule.

### CODE REVIEW

File: bench/vibrato-bench/Cargo.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-vibrato = { path = "../../thirdparty/vibrato/vibrato", default-features = false, features = ["unchecked"] }
+vibrato = { path = "../../thirdparty/vibrato/vibrato" }
```
Comment (vbkaisetsu) [inline bench/vibrato-bench/Cargo.toml]: Why use git submodule?
```suggestion
vibrato = "=0.3.0"
```

Comment (kampersanda) [reply bench/vibrato-bench/Cargo.toml]: To produce system dictionaries, we employ script files in `thirdparty/vibrato/scripts`. So, we have to install the git submodule. To keep consistency with the produced dictionaries, we specify the dependency to the git submodule.

**Comment by kampersanda** [comment]
@vbkaisetsu I am going to update vibrato to distribute precompiled models. After it will be completed, this PR will be modified accrodingly.

**Comment by kampersanda** [comment]
@vbkaisetsu I updated this PR according to the release of Vibrato v0.3.1. Could you review it?

**Review by vbkaisetsu — APPROVED** [review]

**Pull request merged**